api_tests/lib_sdk/lnd.ts

```diff
@@ -131,8 +133,26 @@ export class Lnd {
         await sleep(1);
     }
 
-    public async fund() {
-        await sleep(1);
+    public async createChainAddress(): Promise<string> {
```

Comment:
My understanding is that this is needed to open the channel later?

Reply:
It's needed to mint.